apps/web/src/app/(private)/layout.tsx

```diff
@@ -26,6 +26,8 @@ export default async function PrivateLayout({
   if (!data.session) return redirect(ROUTES.auth.login)
 
   const { workspace, user } = await getCurrentUser()
+  if (!user) return redirect(ROUTES.auth.login)
```

Comment:
Without session there is no user. But doesn't hurt this check

Reply:
this is why I mentioned to do the check in a middleware as redirects in pages always throw. Nextjs docs actually recommend doing any pre-render redirects in middleware (for the future)